sdk/tables/Azure.Data.Tables/tests/TableServiceLiveTestsBase.cs

```diff
@@ -97,6 +82,18 @@ public async Task TablesTestSetup()
                 _ => throw new NotSupportedException("Unknown endpoint type")
             };
 
+            ConnectionString = _endpointType switch
```

Comment:
Does anything use this?

Reply:
Not currently - I used it to temporarily test that all tests would pass using a ctor using that as the argument, rather than the shared key cred.